# 简化云台控制器配置文件
simplified_gimbal_controller:
  ros__parameters:
    # 图像参数
    image_width: 640
    image_height: 480
    camera_horizontal_fov: 53.35  # 度
    camera_vertical_fov: 41.30    # 度
    use_camera_info: true
    
    # PID参数
    kp_yaw: 2.0
    ki_yaw: 0.1
    kd_yaw: 0.5
    kp_pitch: 2.0
    ki_pitch: 0.1
    kd_pitch: 0.5
    
    # 控制参数
    dead_zone_pixels: 10          # 死区像素数
    max_angle_step: 2.0          # 最大角度步长(度)
    yaw_limit_min: -90.0         # 偏航角最小值(度)
    yaw_limit_max: 90.0          # 偏航角最大值(度)
    pitch_limit_min: -30.0       # 俯仰角最小值(度)
    pitch_limit_max: 30.0        # 俯仰角最大值(度)
    
    # 跟踪参数
    tracking_enabled: true        # 是否启用目标跟踪
    control_frequency: 20.0      # 控制频率(Hz)
